src/test/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesSpec.scala

```diff
@@ -199,7 +199,34 @@ class UserRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest {
     }
   }
 
+  "DELETE /api/admin/user/{userSubjectId}/petServiceAccount" should "delete a pet service account for a user" in withAdminRoutes { (samRoutes, adminRoutes) =>
+    // create a user
+    Post("/register/user") ~> samRoutes.route ~> check {
+      status shouldEqual StatusCodes.Created
+      responseAs[UserStatus] shouldEqual UserStatus(UserStatusDetails(defaultUserId, defaultUserEmail), Map("ldap" -> true, "allUsersGroup" -> true, "google" -> true))
+    }
 
+    // create a pet service account
```

Comment:
I would like one more create and delete

Reply:
Create and delete pet you mean? Ok I can add